### DISCUSSION TITLE
Obtaining encryption key from Barnes and Noble app

### DISCUSSION THREAD

**Discussion by Kweap** [noDRM/DeDRM_tools#518] [Q&A]
So, I'm very new to doing this, don't know much. I have been able to dedrm my kindle books just fine but I'm a little stumped on how to get my Barnes and Noble books dedrmed. I think I need an encryption key which can be obtained from the Barnes and Noble app. the pc version doesn't exist for download anymore so I am wondering what are the steps do I have to take to obtain the key? I have the android version installed both on an emulator and my actual phone. Thank you for reading and thank you in advance for the help if ya help :)

Also I'm _pretty_ sure I know what to do with the key once I have it but I would just like to be sure, so if someone could also explain what to do once I have it that would be greatly appreciated!

**Comment by ElleKayEm** [comment]
Here are a couple how-to write-ups on this:
https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/
https://kmhilton.wordpress.com/2021/05/25/dedrm-your-nookbooks-for-dummies/

This process worked for others in the past.  I have not heard any reports in a while about whether or not it is still working.  I liberated my nookbooks years ago before it became so difficult and have not bought any since.

**Comment by charles-slc** [comment]
I've been using the older v10.0.3 version with the Nook App from the Microsoft Windows app store as discussed in this thread:
https://github.com/noDRM/DeDRM_tools/discussions/9
with no issues. I just installed the v10.0.9 version so I could deal with the new Amazon BS (with Kindle v1.40.xxx) and that worked.
I just re-import a B&N book with 10.0.9 and had no issues.
I do wonder why the readme says "These tools no longer work well with books from Barnes & Noble."

**Reply by ElleKayEm** [reply]
If you already have the Windows Nook app, I can see that you'd probably be fine.  However, the Windows Nook app is no longer available.  So you are back to the method described in the links I gave which uses an old Android Nook app on a rooted device or an emulator.  (And I still haven't heard whether it works anymore or not.)  Intial setup is a lot tougher because DeDRM cannot extract the key for you like it could from the Windows app.  If you can manage to get your key and your book files, then DeDRM can still do the actual decryption.

**Reply by ElleKayEm** [reply]
There also used to be Windows and Mac desktop Nook programs that DeDRM worked with.  Those apps were cut off from connecting to B&N many years ago.

**Reply by charles-slc** [reply]
@ElleKayEmwow wow didn't realize the Nook for Windows app had been removed (as of 11/29/2023)
So much for building a new PC... :(

**Reply by MegMM** [reply]
Yeah, just discovered that Nook for PC was terminated in November after I bought 2 books today. Canceled my order with B&N, and will probably purchase from Amazon simply because I expect them to be a-holes. So unhappy.
